docs/operation/operation.md

```diff
@@ -926,6 +931,7 @@ The parameters of `-passive-health-check` option are:
 + `min-drop-probabilty=<0.0 <= p <= 1.0>` - the minimum possible probability of unhealthy endpoint being not considered while choosing the endpoint for the given request. The same value is in fact used as minimal failed requests ratio for PHC to be enabled for this endpoint
 + `max-drop-probabilty=<0.0 <= p <= 1.0>` - the maximum possible probability of unhealthy endpoint being not considered
 while choosing the endpoint for the given request
++ `max-unhealthy-endpoints-ratio=<0.0 <= r <= 1.0>` - the maximum ratio of unhealthy endpoints for PHC to try to mitigate ongoing requests
```

Comment:
Maybe we should not use angle brackets `<>` but `[]`. At least in my experience in math we use these which makes more sense to at least me. :)